docs/content/en/docs/getting-started/optional/autoscaling.md

```diff
@@ -35,10 +35,9 @@ Configure an EKS Anywhere worker node group to be picked up by a Cluster Autosca
           machineGroupRef:
             kind: VSphereMachineConfig
             name: worker-machine-b
-          count: 1
 ```
 
-Note that if no `count` is specified for the worker node group it will default to the `autoscalingConfiguration.minCount` value.
+Note that if `count` is specified for the worker node group, it will be ignored during cluster creation as well as cluster upgrade.
```

Comment:
We should talk about the defaulting logic that CAPI follows, which is set it to minCount or something right? And maybe even reference their docs talking about it if possible.

Reply:
Agreed, I will update it